config/locales/en/manageiq.po

```diff
@@ -6895,16 +6903,10 @@ msgstr ""
 msgid "Download the Entire #{@msg_title} Log File"
 msgstr ""
 
-msgid "Edit this #{ui_lookup(:table=>\"middleware_server\")}"
+msgid "Show Capacity & Utilization data for this Server"
 msgstr ""
 
-msgid "Remove this #{ui_lookup(:table=>\"middleware_server\")} from the VMDB"
```

Comment:
are you sure it should be part of thi commit? (and the rest of the translations in different languages)